cms/models.py

```diff
@@ -59,4 +60,60 @@ class ProgramPage(Page):
     content_panels = Page.content_panels + [
         FieldPanel('description', classname="full"),
         FieldPanel('program'),
+        InlinePanel('FAQs', label='Frequently Asked Questions'),
+        InlinePanel('Courses', label='Program Courses')
+    ]
+
+    def get_context(self, request):
+        js_settings = {
+            "gaTrackingID": settings.GA_TRACKING_ID,
+            "host": webpack_dev_server_host(request)
+        }
+
+        context = super(ProgramPage, self).get_context(request)
+
+        context["style_src"] = get_bundle_url(request, "style.js")
+        context["public_src"] = get_bundle_url(request, "public.js")
+        context["style_public_src"] = get_bundle_url(request, "style_public.js")
+        context["authenticated"] = not request.user.is_anonymous()
+        context["username"] = request.user.username
+        context["js_settings_json"] = json.dumps(js_settings)
+        context["title"] = self.title
+
+        return context
+
+
+class Courses(Orderable):
```

Comment:
We should use singular names for models. Can you rename this `Course` and `FrequentlyAskedQuestion`?